server/server_test.go

```diff
@@ -0,0 +1,83 @@
+package server
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [the file doesn't have an appropriate header](https://app.codacy.com/gh/hrple/common/pullRequest?prid=5175046)